test/vtc/news-redirects.vtc

```diff
@@ -0,0 +1,69 @@
+varnishtest "Test new edge redirects"
+
+# App mock server
+server s1 {
+  rxreq
+  txresp -status 404 -body "App backend"
+} -start
+
+# Start varnish with our VCL
+varnish v1 -vcl {
+  vcl 4.1;
+
+  import std;
+
+  # include the separate redirects vcl file
+  include "/etc/varnish/news-redirects.vcl";
```

Comment:
This is a neat trick! It saves on copy-pasting configs between tests and the config that gets deployed. The other tests could do with a similar refactor. Adding this to the post 1.0 list.